pkg/virtctl/vnc/vnc.go

```diff
@@ -82,6 +83,8 @@ func NewCommand() *cobra.Command {
 	}
 	cmd.Flags().StringVar(&listenAddress, "address", listenAddress, "--address=127.0.0.1: Setting this will change the listening address of the VNC server. Example: --address=0.0.0.0 will make the server listen on all interfaces.")
 	cmd.Flags().BoolVar(&proxyOnly, "proxy-only", proxyOnly, "--proxy-only=false: Setting this true will run only the virtctl vnc proxy and show the port where VNC viewers can connect")
+	cmd.Flags().BoolVar(&forceConnection, "force-connection", false,
```

Comment:
```suggestion
	cmd.Flags().BoolVar(&forceConnection, "force", false,
```

`force` suffices IMO